src/main/resources/sql/script.sql

```diff
@@ -0,0 +1,26 @@
+SELECT * FROM dealerships;
+
```

Comment:
Are these like test queries to see what was in your table?

Reply:
yes, one of the workbooks asked us to do a handful of queries and save them in a sql file.

Reply:
Perfect.